Restructure Source Maps Upload page (#5799)
getsentry/sentry-docs: src/platform-includes/sourcemaps/upload/cli/javascript.mdx

> When choosing [`sentry-cli`](/product/cli/) as the method to upload your source files, we assume you already know how to generate source maps within your build tool chain.

Reviewer: ```suggestion
If you choose [`sentry-cli`](/product/cli/) as the method to upload your source files, we assume you already know how to generate source maps within your build tool chain.
```

Gotta get the "you" in there or we're the ones doing the choosing.

Sentry uses [releases](/product/releases/) to match the correct uploaded source maps to your events. You can use `sentry-cli` to create releases and upload source maps to Sentry.

## 1. Update SDK Options

To be able to match Sentry errors with the correct release, you must provide a `release` property in the SDK options:

```javascript
Sentry.init({
  // Make sure this value is identical to the name you give the release that you

>   // will create below using Sentry CLI

Reviewer: ```suggestion
  // create below using Sentry CLI
```

  release: "<release_name>",
});
```

## 2. Upload Source Maps Using Sentry CLI

<Note>

You can find instructions on how to install Sentry CLI here: https://docs.sentry.io/product/cli/installation/

For more info on Sentry CLI configuration visit the [Sentry CLI configuration docs](/product/cli/configuration/).

</Note>

Make sure Sentry CLI is configured for your project. You can use a `.sentryclirc` file or environment variables:

<Note>

Visit the [auth token user settings page](https://sentry.io/settings/account/api/auth-tokens/) to find your auth tokens.

</Note>

```ìni {filename:.sentryclirc}
[auth]
token=your-auth-token

[defaults]
org=___ORG_SLUG___
project=___PROJECT_SLUG___
```

```bash
export SENTRY_AUTH_TOKEN=your-auth-token
export SENTRY_ORG=___ORG_SLUG___
export SENTRY_PROJECT=___PROJECT_SLUG___
```

Then, create a release with Sentry CLI. The release name must be unique within your organization:

```bash
sentry-cli releases new <release_name>
```

Next, upload your release artifacts (source files and source maps):

```bash
sentry-cli releases files <release_name> upload-sourcemaps /path/to/files
```

<Note>

> You can verify your uploaded artifacts by navigating to **[Project] > Project Settings (gear icon in top right corner) > Source Maps** on Sentry.

Reviewer: ```suggestion
You can verify that your upload was successful by navigating to **[Project] > Project Settings (gear icon in top right corner) > Source Maps** on Sentry.
```

FWIW, if we send them via Releases -> specific release -> Source Maps (bottom of RH sidebar), it's 3 clicks rather than 4. Up to you.

</Note>

> Until now, the release is in a draft state (“_unreleased_”). Once all source maps have been uploaded and your app has been published successfully, finalize the release with the following command:

Reviewer: ```suggestion
At this point, the release is in a draft state (“_unreleased_”). Once all source maps have been uploaded and your app has been published successfully, finalize the release with the following command:
```
